client/src/lib/TinyMCEActionRegistrar.js

```diff
@@ -27,6 +32,42 @@ class TinyMCEActionRegistrar {
   getActions(menu) {
     return this.actions[menu] || [];
   }
+
+  /**
+   * Registers a new map item for command name and url test
+   *
+   * @param {String} command
+   * @param {RegExp} test
+   * @return {TinyMCEActionRegistrar}
+   */
+  addCommandWithUrlTest(command, test) {
+    this.editorCommandsToUrlTestsMap[command] = test;
+    return this;
+  }
+
+  /**
+   * Gets an editor command name based on the given url
+   *
+   * @param {String} url Inserted url
+   * @return {String} Editor command name
+   */
+  getEditorCommandFromUrl(url) {
+    let command = 'sslinkexternal';
```

Comment:
could we have a `getDefaultCommand`? So this could be tweaked/replaced in future?